sogou-search-api.gemspec

```diff
@@ -23,8 +23,9 @@ Gem::Specification.new do |spec|
 
   spec.add_runtime_dependency 'savon', '~> 2.12'
 
-  spec.add_development_dependency 'bundler', '~> 1.14'
-  spec.add_development_dependency 'rake', '~> 10.0'
-  spec.add_development_dependency 'rspec', '~> 3.0'
-  spec.add_development_dependency 'simplecov', '~> 0.14'
+  spec.add_development_dependency 'bundler', '~> 2.1'
+  spec.add_development_dependency 'rake', '~> 13.0'
+  spec.add_development_dependency 'rspec', '~> 3.11'
+  spec.add_development_dependency 'simplecov', '~> 0.21'
+  spec.add_development_dependency 'gem-release', '~> 2.2'
```

Comment:
I understand this is in order to push a new tag and create GitHub release? I am not sure if we should be adding this to the gemspec. Maybe best if it was added to the Gemfile?

Doesn't bundler's `rake release` work here?